doc/modules.asciidoc

```diff
@@ -6,6 +6,7 @@ Aligns its children to the given faces.
 Parameters
 ++++++++++
 |=========
+|vector|alignment in +/- x, y, z
```

Comment:
The actual name of the parameter is `anchor`
https://github.com/GilesBathgate/RapCAD/blob/e9a1a0e2ceceeea2a3e30934e3d4d407ac9a519d/src/module/alignmodule.cpp#L41

Reply:
Thanks for pointing this out. I updated the pull request with the correct terminology. I hope you don't mind me adding this.